.travis.yml

```diff
@@ -1,19 +1,18 @@
 # GH_TOKEN and NPM_TOKEN are expected to be set
 # in https://travis-ci.org/apiaryio/gavel.js/settings
-language: "node_js"
+language: 'node_js'
 node_js:
-  - "6"
-  - "8"
-  - "10"
+  - '8'
+  - '10'
```

Comment:
Quotes changes triggered by Prettier.
I'd like to introduce it to the pipeline anyway.

Reply:
Prettier touches YML? 🤔